token-validation-spring-demo/src/test/java/no/nav/security/token/support/demo/spring/LocalSecurityConfiguration.java

```diff
@@ -0,0 +1,12 @@
+package no.nav.security.token.support.demo.spring;
+
+import no.nav.security.token.support.spring.test.EnableMockOAuth2Server;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+
+@Configuration
+@Profile("local")
+@EnableMockOAuth2Server
+public class LocalSecurityConfiguration {
+
```

Comment:
Why an empty class?

Reply:
In a normal app this class would contain some configuration, created here to show usage of `@EnableMockOAuth2Server` and @Profile annotations.